src/MediaPlayer/MediaPlayer.cs

```diff
@@ -18,6 +18,7 @@ namespace XamCore.MediaPlayer {
 	[Native]
 	[NoMac]
 	[NoTV]
+	[Availability (Deprecated = Platform.iOS_9_0)]
```

Comment:
The use of the enums can't be replaced with `AVPlayerViewController`. However wherever the enums are used (e.g. assigned to a property properties) it's with something also deprecated that will have the message.